browser/speedreader/speedreader_tab_helper.h

```diff
@@ -73,12 +77,19 @@ class SpeedreaderTabHelper
   // returns nullptr if no bubble currently shown
   SpeedreaderBubbleView* speedreader_bubble_view() const;
 
+  // returns nullptr if no bubble currently shown
+  views::BubbleDialogDelegateView* speedreader_webui_bubble_delegate_view()
+      const;
+
   // Displays speedreader information
   void ShowSpeedreaderBubble();
 
   // Displays reader mode information
   void ShowReaderModeBubble();
 
+  // Displays speedreader UI controls
+  void ShowSpeedreaderWebUIBubble();
```

Comment:
Looks like it should be private.

Reply:
why private? this can potentially be used by icon button view interface to be able call on button click. 

Currently, all of the click logic is baked inside this tab helper as ``::ProcessIconClick` however we have plans to consolidate all of this into one bubble and icon only.